src/main/java/org/dbpedia/util/DBpediaUriEncode.java

```diff
@@ -0,0 +1,63 @@
+/**
+ *  Copyright 2015 DBpedia Spotlight
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+package org.dbpedia.util;
+
+//import org.dbpedia.spotlight.util.WikiUtil;
+
+import org.dbpedia.extraction.util.WikiUtil;
+
+import java.io.Serializable;
+import java.util.regex.Pattern;
+/*
+Class used to Encode wiki Uri
+ */
+public class DBpediaUriEncode {
```

Comment:
This one was in pignlproc for computing the dbpedia uri.. So I just copied this from there.